azure-pipelines.yml

```diff
@@ -131,6 +117,12 @@ stages:
           path: analyzers/its/binaries/
           artifact: Binaries
 
+      - task: PublishPipelineArtifact@1
```

Comment:
_Unrelated thing that I've just randomly noticed:_ Line 5 - it's not `midnight` anymore 😃

Reply:
Maybe `Nightly build` instead?

Reply:
It's a daily build since it runs at 7 our time 😄

Reply:
Single-digit hours are `night` for me 😆